clouddriver-eureka/src/main/groovy/com/netflix/spinnaker/clouddriver/eureka/deploy/ops/AbstractEurekaSupport.groovy

```diff
@@ -149,26 +154,10 @@ abstract class AbstractEurekaSupport {
             throw new RetryableException("Non HTTP 200 response from discovery for instance ${instanceId}, will retry (attempt: $retryCount}).")
           }
         }
-      } catch (RetrofitError retrofitError) {
-        if (discoveryStatus == DiscoveryStatus.Disable) {
-          def alwaysSkippable = retrofitError.response?.status == 404
-          def willSkip = alwaysSkippable || !strict
-          def skippingOrNot = willSkip ? "skipping" : "not skipping"
-
-          String errorMessage = "Failed updating status of ${instanceId} in application $applicationName in discovery" +
-            " and strict=$strict, $skippingOrNot disable operation."
-
-          // in strict mode, only 404 errors on disable are ignored
-          if (!willSkip) {
-            errors[instanceId] = retrofitError
-          } else {
-            skipped.add(instanceId)
-          }
-
-          task.updateStatus phaseName, errorMessage
-        } else {
-          errors[instanceId] = retrofitError
-        }
+      } catch (NotFoundException e) {
```

Comment:
@robzienert because of the special error handling in this class, I'm not actually sure catching the new error types improves the readability; let me know if you'd prefer I leave this file as-is.